tests/unit/test_factory/test_router.py

```diff
@@ -6,6 +6,37 @@ def test_path_and_component(self):
         route = VueRoute("/path", VueComponent)
         assert route == {"path": "/path", "component": VueComponent.init_dict()}
 
+    def test_path_and_components(self):
+        route = VueRoute(
+            "/path", components={"default": VueComponent, "named": VueComponent}
+        )
+        assert route == {
+            "path": "/path",
+            "components": {
+                "default": VueComponent.init_dict(),
+                "named": VueComponent.init_dict(),
+            },
+        }
+
+    def test_path_and_components_and_children(self):
+        route = VueRoute(
+            "/path",
+            VueComponent,
+            children=[VueRoute("/path", VueComponent), VueRoute("/path2", VueComponent),],
```

Comment:
```suggestion
            children=[
                VueRoute("/path", VueComponent),
                VueRoute("/path2", VueComponent),
            ],
```
fixes formatting issues